src/components/card-list/card-list.component.jsx

```diff
@@ -1,10 +1,38 @@
-import React from 'react';
+import React, { createContext, useState } from 'react';
 import './card-list.styles.css';
 import { Card } from '../card/card.component'
-
+import PokeInfo from '../solo-card/Poke-info';
+import {useNavigate } from 'react-router-dom';
 export const CardList  = props => {
-    return <div className="card-list">{
+    const [user, setUser] = useState({"yo":"hehe"});
+    const UserContext = createContext()
+  const navigate=useNavigate();
+    const handleAction=async(ele)=>{
+        const response=await fetch(ele);
+const data=await response.json();

```

Comment:
Use prettier or eslint

Reply:
is it for format document or something else?

Reply:
Yes for formatting.

Reply:
I can do that using vs code formatting options i guess.